File: PageBook/Models/ToDoModel.cs
using SQLite;

namespace PageBook.Models;

public class ToDoModel
{
    [PrimaryKey]
    public string Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime ModifiedAt { get; set; }
    public bool IsDo { get; set; }
    public string Tags { get; set; }
 }

File: app/src/main/java/ru/maxgog/pagebook/models/TodoModel.kt
package ru.maxgog.pagebook.models

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "todos")
data class TodoModel(
    @PrimaryKey(autoGenerate = true) val id: Int = 0,
    @ColumnInfo(name = "Title") val title: String,
    @ColumnInfo(name = "Description") val description: String?,
    @ColumnInfo(name = "IsCompleted") val isCompleted: Boolean = false,
    @ColumnInfo(name = "AtCreated") val createdAt: Long = System.currentTimeMillis(),
    @ColumnInfo(name = "HasReminder") val hasReminder: Boolean = false,
    @ColumnInfo(name = "ReminderTime") val reminderTime: Long? = null,
    @ColumnInfo(name = "IsAddedToCalendar") val isAddedToCalendar: Boolean = false
)